src/heading.js

```diff
@@ -31,7 +28,7 @@ export default class Heading extends Plugin {
 	 * @inheritDoc
 	 */
 	static get requires() {
-		return [ Paragraph, HeadingEngine ];
```

Comment:
I expected that `Paragraph` will be required in `Editing` but it isn't. Why is it not needed anymore? Isn't it required when applying "default option" (returning from heading to paragraph)?

Reply:
If it's not then it looks like bug.

Reply:
Ill take care of this.

Reply:
`Paragraph` is required by `HeadingEditing`, so 👍. I got confused cause it has not been shown on diff.

Reply:
Oh, OK Then :D